src/PaymentBundle/Resources/docs/reference/configuration.rst

```diff
@@ -117,7 +117,7 @@ Full Configuration Options
         selector: sonata.payment.selector.simple
 
         # service which generate the correct order and invoice number
-        generator: sonata.payment.generator.mysql
+        generator: sonata.payment.generator.mysql # or sonata.payment.generator.postgres
```

Comment:
And could you add a configuration Test? I think @greg0ire did this in another PR